content/22.vaccines.md

```diff
@@ -304,22 +304,67 @@ Limited distribution was shared with the UAE [@url:https://www.nytimes.com/inter
+Similarly to the SARS-CoV vaccine candidates, the MERS-CoV protein subunit vaccine candidates generally target the RBD [@doi:10.3389/fmicb.2020.00298; @doi:10.1016/j.vaccine.2006.10.031; @doi:10.1016/j.ebiom.2015.08.031; @doi:10.1371/journal.pone.0112602; @doi:10.1016/j.vaccine.2018.02.065; @doi:10.1016/j.virol.2016.10.005], with some targeting the full length S protein [@doi:10.1073/pnas.1707304114], and non RBD protein fragments such as the SP3 peptide [@doi:10.1089/vim.2014.0080] and the recombinant N-terminal domain (rNTD) [@doi:10.1016/j.vaccine.2016.11.064].
+No protein subunit vaccine for MERS-CoV has progressed beyond preclinical research to date. 
+
+More than 20 protein subunit vaccines have entered clinical trials for COVID-19 since the beginning of the pandemic by companies such as Sanofi and GlaxoSmithKline, Nanogen, and the Serum Institute of India [@url:https://covid19.trackvaccines.org/vaccines/].
+However, the NVX-CoV2373, produced by U.S. company Novavax and partners has had the most success to date. 
+NVX-CoV2373 is a protein nanoparticle vaccine candidate against SARS-CoV-2 that is constructed from a mutated prefusion SARS-CoV-2 spike protein in combination with a specialized saponin-based adjuvant to elicit an immune response against SARS-CoV-2. 
+The spike protein is recombinantly expressed in Sf9 insect cells [@doi:10.1038/s41467-020-20653-8], which have previously been used for several other FDA-approved protein therapeutics [@doi:10.1021/acs.iecr.8b00985] and contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) along with two proline substitutions (K986P and V987P) that improve thermostability [@doi:10.1038/s41467-020-20653-8]. 
+In preclinical mouse models, Novavax-CoV2373 elicited high anti-spike IgG titers 21 to 28 days post-vaccination that could neutralize the SARS-CoV-2 virus and protect the animals against virus challenge, with particularly strong effects when administered with the proprietary adjuvant Matrix-M^TM^ [@doi:10.1038/s41467-020-20653-8]. 
+In a phase I/II trial, a two-dose regimen of NVX-CoV2373 was found to induce anti-spike IgG levels and neutralizing antibody-titers exceeding those observed in convalescent plasma donated by symptomatic patients [@doi:10.1056/NEJMoa2026920]. 
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response. 
+
+In a phase III randomized, observer-blinded, placebo-controlled clinical trial in 14,039 participants, two 5-μg doses of NVX-CoV2373 or placebo were administered 21 days apart in a 1:1 ratio from late September to late November 2020 [@doi:10.1056/NEJMoa2107659].
+The primary endpoint of the trial was to prevent PCR-confirmed symptomatic mild, moderate or severe COVID-19 at least 7 days after the second dose.
```

Comment:
For an endpoint, I think the endpoint would be the development of the disease, not the prevention, right? (You know a lot more about clinical trials than I do, but that was my understanding!)

Reply:
hmmm... the paper says the following "The primary end point was the efficacy of the NVX-CoV2373 vaccine against the first occurrence of virologically confirmed symptomatic mild ... maybe it would be best to reword the same way. It is prevention, but not in the sense that it is defined in the paper.

Reply:
Agreed, probably the rush to publish!